### PR TITLE
Enable aggregation layer for k8s api server

### PR DESCRIPTION
**Pull request by pipo02mix**
Towards https://github.com/giantswarm/giantswarm/issues/2656

### CODE REVIEW

**Comment by pipo02mix** [comment]
Tested PTAL

File: vendor/github.com/giantswarm/k8scloudconfig/CHANGELOG.md [hunk 1]
```diff
@@ -11,6 +11,9 @@ version directory, and then changes are introduced.
 
 ## [v3.2.1] WIP
 
+### Changed
+- Enable aggregation layer in Kubernetes API server
```
Comment (xh3b4sd) [inline vendor/github.com/giantswarm/k8scloudconfig/CHANGELOG.md]: Just FYI. IMO it would be nice in the future to track the PR links here.

Comment (pipo02mix) [reply vendor/github.com/giantswarm/k8scloudconfig/CHANGELOG.md]: can you give me an example?

Comment (xh3b4sd) [reply vendor/github.com/giantswarm/k8scloudconfig/CHANGELOG.md]:
```
- Enable aggregation layer in Kubernetes API server (https://github.com/giantswarm/k8scloudconfig/pull/322)
```

**Pull request merged**